ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -707,6 +713,7 @@ public class StoreConfig {
   public final String storeBootstrapInProgressFile;
 
   public StoreConfig(VerifiableProperties verifiableProperties) {
+    fileCopyFeatureEnabled = verifiableProperties.getString(FILECOPY_FEATURE_ENABLED, "false");
```

Comment:
Please use this from server config: https://github.com/linkedin/ambry/pull/2966/files#diff-301f6df5395b68a10502c0c45fa96d90cb052793a96c1569c6890b07185f5aeb